benchmarking/src/llmperf/models.py

```diff
@@ -7,7 +7,7 @@ class RequestConfig(BaseModel):
 
     Args:
         model: The model to use.
-        prompt: The prompt to provide to the LLM API.
+        prompt_tuple: A tuple containing the prompt to provide to the LLM API along with the tokenized prompt length.
```

Comment:
Renaming this argument for more clarity. The input itself is a tuple containing the prompt rather than a prompt. Makes it more clear for the developer and user!